resources/osl_mongodb.rb

```diff
@@ -0,0 +1,30 @@
+resource_name :osl_mongodb
+provides :osl_mongodb
```

Comment:
Do this instead of the if statement at the end of the action block.
```suggestion
provides :osl_mongodb, platform: 'redhat'
```